backpack/core/derivatives/rnn.py

```diff
@@ -248,16 +229,15 @@ def _weight_ih_l0_jac_t_mat_prod(
             product
         """
         self._check_parameters(module)
-        N_axis, _ = self.get_batch_and_time_axes(module)
         return einsum(
-            f"vtnh,{'nt' if module.batch_first else 'tn'}j->v{'' if sum_batch else 'n'}hj",
+            f"vtnh,ntj->v{'' if sum_batch else 'n'}hj",
```

Comment:
Re `a_jac_t`: Internal `transpose` in `einsum` because first operand is `tn`, second is `nt`